### PR TITLE
Add new product update

### PR DESCRIPTION
**Pull request by mcclowes**
# Description

New post

### CODE REVIEW

File: blog/230404-product-update.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Product update #4"
+date: "2023-04-04"
+tags: ["Deprecation"]
+authors: mcclowes
+---
+
+We've improvemed our data type coverage for a number of integrations.
```
Comment (d-laing) [inline blog/230404-product-update.md:8]:
```suggestion
We've improved our data type coverage for a number of integrations.
```

File: blog/230404-product-update.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+### Integration coverage improvements
+
+#### QuickBooks Online
+- Customer tracking on Expense lines (Direct Cost - Pull and Push)
+- Push Payment on Account refunds
+
+#### MYOB 
+- Push Journal Entries
+- Pull 'Jobs' on Tracking Categories
```
Comment (d-laing) [inline blog/230404-product-update.md:20]:
```suggestion
- Pull "Jobs" on Tracking Categories
```

File: blog/230404-product-update.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+#### Freshbooks 
+- Pull Bills
+- Pull Bill Payments
+
+#### FreeAgent
+- We've added support for Pushing Direct Cost attachments
```
Comment (d-laing) [inline blog/230404-product-update.md:27]:
```suggestion
- We've added support for pushing Direct Cost attachments
```

File: blog/230404-product-update.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## Deprecations
+
+Don't forget about these immenent deprecations:
```
Comment (d-laing) [inline blog/230404-product-update.md:31]:
```suggestion
Don't forget about these imminent deprecations:
```

**Review by d-laing — CHANGES_REQUESTED** [review]
Great to highlight these new features!

File: blog/230404-product-update.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Product update #4"
+date: "2023-04-04"
+tags: ["Deprecation"]
```
[the remark this answers is not in the document]
Comment (d-laing) [reply blog/230404-product-update.md:4]: Just checked: should be `Product` and `Update`.

File: blog/230404-product-update.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+## Deprecations
+
+Don't forget about these immenent deprecations:
+
+- <https://docs.codat.io/updates/230411-api-keys>
+- <https://docs.codat.io/updates/230411-deletion-of-data>
+- <https://docs.codat.io/updates/230411-legacy-bank-account-endpoints>
+
+The next round of deprecations will be in July:
```
Comment (pzaichkina) [inline blog/230404-product-update.md:37]: I guess this now also needs to have the Assess deprecation

File: blog/240101-draft-update.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Upcoming YYYY-MM-DD: Title"
+date: "2020-01-01"
+tags: ["Deprecation"]
```
Comment (pzaichkina) [inline blog/240101-draft-update.md:4]: yeah the tag should probably be Product and Update

**Pull request merged**